seqr/views/apis/family_api.py

```diff
@@ -45,9 +45,22 @@ def family_page_data(request, family_guid):
     sample_models = Sample.objects.filter(individual__family=family)
     samples = get_json_for_samples(sample_models, project_guid=project.guid, family_guid=family_guid, skip_nested=True, is_analyst=is_analyst)
     response = {
-        'samplesByGuid': {s['sampleGuid']: s for s in samples},
+        'samplesByGuid': {s['sampleGuid']: {**s, 'rnaSeqTypes': []} for s in samples},
     }
 
+    # Add Rna Seq metadata to samples
+    rna_seq_models = [
```

Comment:
I'm thinking this should belong in `get_json_for_samples()` because I also need to update the `project_overview` api if I also want the Family Table sample to have the same hover-over behavior as the Family Page sample.

Reply:
So the Family Table sample in the compact summary view should only be a search sample - if its not thats a bug. When you expand the Family row to see the full detail it loads the same data as when you load the family page directly so there shoudl be no issue there. So you should only need to change behavior in one place